docs/api/annotations.md

```diff
@@ -0,0 +1,233 @@
+# Annotations
+
+An annotation assigns pronunciation errors to specific fragments of an audio recording performed the
+an annotator user.
+
+## List all annotations
+
+A user with the 'ADMIN' role will receive all the annotations.
+Non-admins will receive only the annotations assigned to them.
+
+### URL
+
+```http
+GET /annotations HTTP/1.1
+```
+
+### Request parameters
+
+Name          | Type     | Description
+--------------|----------|------------
+`annotator`   | `string` | **Optional** The unique identier of the annotator to find the annotations for.
+`assignment`  | `string` | **Optional** The assignment for which to find the annotations.
+`recording`   | `string` | **Optional** The speech recording for which to find the annotations.
+`done`        | `bool`   | **Optional** Whether or not and how to filter on the done property. None when all values are allowed, True if only done annotations are desired. False for all not finished annotations.
+`perPage`     | `string` | **Optional** Amount of annotations to return.
+`startCursor` | `string` | **Optional** A previously returned cursor to use as an 'offset'.
+
+```http
+GET /annotations HTTP/1.1
+Accept: application/json
+```
```

Comment:
What is this example block about?

Reply:
Since most of are filter params, there's not much to show,

Reply:
Well, we were describing filters a different way: https://itslanguage.github.io/itslanguage-docs/master/api/users.html#filters